### DISCUSSION TITLE
v.0.2.0-alpha5 - Zotero 6 compatibility

### DISCUSSION THREAD

**Discussion by argenos** [argenos/zotero-mdnotes#165] [General]
## What's Changed

- Update filepicker imports to use `zotero/modules` (#126)
- Upgrade max version for compatibility with Zotero 6 (#133, #77)
* Docs: Update formatting.md in docs by @einalex in https://github.com/argenos/zotero-mdnotes/pull/89
* Docs: Clearer text by @Jmuccigr by https://github.com/argenos/zotero-mdnotes/pull/93
* Docs: fix error in formatting.md by @atraphaxis in https://github.com/argenos/zotero-mdnotes/pull/159
* Docs: replace {{noteTitle}} with {{title}} by @Glunk in https://github.com/argenos/zotero-mdnotes/pull/145

## New Contributors
* @einalex made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/89
* @Jmuccigr made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/93
* @atraphaxis made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/159
* @Glunk made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/145

**Full Changelog**: https://github.com/argenos/zotero-mdnotes/compare/0.2.0-alpha4...0.2.0-alpha5

<hr /><em>This discussion was created from the release <a href='https://github.com/argenos/zotero-mdnotes/releases/tag/0.2.0-alpha5'>v.0.2.0-alpha5 - Zotero 6 compatibility</a>.</em>

**Comment by AmiaiBai** [comment]
Why can't I choose Export directory and Template foder?
